index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/8d0a2aa6d4.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <title>first capstone project</title>
+</head>
+<body>
+    <header> 
+        <div class="socials_container">
+            <div class="socials_wrapper">
+               <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <ul>
+                    <a href="#"><li>English</li></a>
+                    <a href="#"><li>My Page</li></a>
+                    <a href="#"><li>Logout</li></a>
+                </ul>
+            </div>
+        </div>
+        <nav>
+            <a href="index.html" class="music_logo">
+                <img src="images/label.png" alt="logo image">
+             </a>
+            <div class="hamburger_menu">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <ul class="mobile_menu">
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#">concert</a></li>
+                <li><a href="#">ticket</a></li>
+                <li><a href="#">shows</a></li>
+                <li><a href="#">News</a></li>
+                <li><a class="nav_link_ticket" href="#"> Get ticket</a></li>
+            </ul>
+            <ul class="nav_link">
+                <li><a class="nav_link_line" href="#">About</a></li>
+                <li><a class="nav_link_line" href="#">concert</a></li>
+                <li><a class="nav_link_line" href="#">ticket</a></li>
+            </ul>
+        </nav>
+
+    </header>
+    <main>
+        <section class="homepage">
+            <div class="homepage_content">
+                <div class="header_content">
+                    <h3>"welcome  y'all"</h3>
+                    <h1 class="h1_image">Monsters of rock world concert 2023</h1>
+                </div>
+                <div class="paragraph">
+                    <p>A great concert celebrating multiple talents from various counties <br>  
+                        which is taking place in gotham city on 15th october</p>
+                </div>
+                <div class="paragraph_two">
+                    <h2>2023.10.15(THU) ~ 16(FRI)</h2>
+                    <p>@lugogo cricket oval </p>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Good job styling your home page :+1:, but as requested by the previous reviewer, please make sure to update the space between `homepage` section and `work_section` to make it looks more like the required template :100:. Below you can see the differences between them.

**Your design**

![Screenshot from 2023-02-10 08-05-15](https://user-images.githubusercontent.com/105117832/218101290-c659adff-d544-4b36-bba7-c4ab4f4ae8e1.png)

**Template**

![Screenshot from 2023-02-10 08-06-31](https://user-images.githubusercontent.com/105117832/218101337-69cc65c5-8977-4832-a043-3f3b75ca4a65.png)

Reply:
- Thanks for the review @IngDavidTM 
- the space between `homepage` section and `work_section` fixed ✅ 
- 
<img width="1440" alt="Screenshot 2023-02-10 at 17 01 19" src="https://user-images.githubusercontent.com/111200646/218110966-7911a0f9-aa4d-4688-b4be-46c5ccbfeef2.png">